_posts/wookiemage/2016-1-14-Colin's Turtles.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: wookiemage
+title: Colin's Turtles
+---
+
+Here's the first Turtle:
+<iframe src="https://trinket.io/embed/python/332d11321e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+This turtle was basically the same turtle I shared in class. It does prompt for info though.
+
+Here's my super Turtle:
+<iframe src="https://trinket.io/embed/python/7c81b5cc31" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+This turtle took it a little further. It not only asks for color options but also asks the user what kind of polygon they want to draw. There's a little math involed.
+```
+#The following lines do simple math to determine the interior angle of the polygon with the number of sides the user has specified.
+#This spans multiple lines because I am unsure of order of operations in python. This declares far more variables than would be needed
+#otherwise.
+sides_2 = sides - 2
+sides_3 = sides_2 * 180
+sides_4 = sides_3 / sides
+turn = 180 - sides_4
+```
+And it involved the use of a loop:
+'''
+#This makes tina walk and turn as many times as there are sides.
+for x in range(0, sides):
+  tina.left(turn)
+  tina.forward(100)
+  '''
```

Comment:
this teenie tiny space in front of the backticks makes Jekyll think you mean to write text instead of end the code block.
You can use the Preview tab in the editor to check the basics of what your post will look like.  That should help you catch small formatting things.
![screenshot 2016-01-15 at 10 01 05 am](https://cloud.githubusercontent.com/assets/1702745/12356140/f80974aa-bb6e-11e5-998d-28bb44928345.png)